python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/test_resources.py

```diff
@@ -1,18 +1,15 @@
-import uuid
-
 import responses
 from dagster_fivetran import FivetranWorkspace
 
 
 def test_basic_resource_request(
+    api_key: str,
+    api_secret: str,
     connector_id: str,
     destination_id: str,
     group_id: str,
```

Comment:
yea these all feel like they could be constants and then the test feels a little less overwhelming, lol.